services/todo-legacy/test/controller/todo-controller.test.ts

```diff
@@ -0,0 +1,121 @@
+import 'mocha';
+import * as _ from 'lodash';
+import { expect, sinon } from '@loopback/testlab';
+import { TodoController } from '../../controllers/todo-controller';
+import { TodoRepository } from '../../repositories/todo';
+
+describe('TodoController', () => {
+  let controller = new TodoController();
+  let fakeData = [
+      {
+        id: 1,
+        title: 'test1',
+        body: 'test1',
+      },
+      {
+        id: 2,
+        title: 'test2',
+        body: 'test2',
+      },
+  ];
+  describe('getTodo', () => {
+    it('returns all todos when called without ID', async () => {
+      let stub = sinon.createStubInstance(TodoRepository);
+      controller.repository = stub;
```

Comment:
Have you considered extracting these two lines into a shared method that can be called from a `beforeEach` hook?